djoser/locale/cs/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,131 @@
+# Copyright (C) Sunscrapers
+# This file is distributed under the same license as the djoser package.
+#
+# Translators:
+# Bertram Bühner <github@sheepyhollow.de>, 2020
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: djoser\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-01-13 17:40+0100\n"
+"PO-Revision-Date: \n"
```

Comment:
Shouldn't it match `POT-Creation-Date`? Weird that's not been autogenerated with `PO-Revision-Date`.